workspaces/tech-insights/.changeset/cuddly-brooms-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-tech-insights-node': minor
```

Comment:
Is it worth bumping these to first major so we can start adhering to proper semver for the changes?

Reply:
Maybe! It's up to the plugin owners I guess. For 0.x though, it's still semantically correct to bump the minor to signify a breaking change.

Reply:
Indeed yeah, I'm just not a fan of abusing the semver system with special cases. For end users it is easier to manage installation and upgrades with a proper baseline versioning.

Let's do it.

Reply:
Sounds good to me - updated!